tsg/ranker/base.py

```diff
@@ -47,6 +47,47 @@ def calculate_query_term_weight(term, query_terms, index_dictionary_path=DICTION
 
     return weight
 
+def conjunctive_score_calc(query_terms, index_dictionary_path= DICTIONARY_PATH, 
+    index_info_path = INDEXINFO_PATH):
+    
+    results_keys = set()
+    terms_lists = {}
+    scores = {}
+    doc_length = {}
+    query_length = {}
+
+    for term in query_terms:
+        terms_lists[term] = get_dictionary_term_list(term, index_dictionary_path)
```

Comment:
doc_list_per_term. sounds horrible.
term_documents is okay. If you want doc_list_FOR_term is also ok